packages/e2e/package.json

```diff
@@ -101,6 +101,7 @@
     "@cardano-sdk/hardware-ledger": "^0.1.0",
     "@cardano-sdk/key-management": "^0.5.1",
     "@cardano-sdk/ogmios": "^0.11.0",
+    "@cardano-sdk/tx-construction": "workspace:^",
```

Comment:
I introduced this. Maybe it should use the version :sweat_smile:  until the other PR is merged